api/src/main/resources/workbench.yaml

```diff
@@ -84,6 +84,140 @@ paths:
           schema:
             $ref: "#/definitions/Profile"
 
+  # Notebook clusters ####################################################################
```

Comment:
I think we need to put some thought into what parts of the notebook service API we want to expose to the client, and remove everything else. There's a lot of stuff here that our UI shouldn't need, and there won't be any other clients of the service.

In general we should consider reworking the API we expose to fit our UX... some thoughts on that below.

If you want to punt on all of this for now, that's totally fine, but let's plan to give it some more thought.

Reply:
I can totally work on fixing this, there's not huge rush in getting notebooks merged, since its sitting behind dev firecloud anyway.